zulipterminal/helper.py

```diff
@@ -275,3 +276,17 @@ def classify_unread_counts(unread_msg_counts: Dict[str, Any])\
         unread_counts['all_msg'] += count
 
     return unread_counts
+
+
+def match_user(user: Any, text: str) -> bool:
+    """
+    Matches if the user full name, last name or email matches
+    with user or not.
+    """
+    keywords = user.caption.split()
+    keywords.append(user.email)
+    keywords.append(user.caption)
```

Comment:
It may be worth adding a comment on why the full name is appended to the list, after starting with the name and splitting it.

Reply:
the keywords contain for name `Aman kumar agrawal`
* Aman
* kumar
* agrawal
* aman kumar agrawal
* user_email

adding full_name helps in further narrowing down the right user. (eg: Aman kumar and aman singh)

Reply:
Yes, I understand that. But, it's not obvious when I looked at the code first. So, the last sentence from the comment above can go as a comment in the code, I think.